ansible/playbook.yml

```diff
@@ -5,13 +5,17 @@
   remote_user: ubuntu
   vars:
     ansible_python_interpreter: auto
+    ansible_ssh_private_key_file: ~/.ssh/shapeworks-ec2-b
```

Comment:
We should document where this ssh file comes from (that is, I assume we created it during EC2 config and then place it here).